datasets/flwr_datasets/partitioner/__init__.py

```diff
@@ -16,9 +16,7 @@
 
 
 from .iid_partitioner import IidPartitioner
+from .natural_id_partitioner import NaturalIdPartitioner
 from .partitioner import Partitioner
 
-__all__ = [
-    "IidPartitioner",
-    "Partitioner",
-]
+__all__ = ["IidPartitioner", "Partitioner", "NaturalIdPartitioner"]
```

Comment:
Given that this list will grow over time, you can add a `,` after the last item and have `black` put them on individual lines